scripts/requirements-fixed.txt

```diff
@@ -56,7 +56,7 @@ mypy==0.800
 mypy-extensions==0.4.3
 naturalsort==1.5.1
 packaging==20.8
-Pillow==8.1.0
+Pillow==8.1.1
```

Comment:
If i'm running `pip-compile` on `sdk-<repo>/scripts/requirements-*.txt` according to http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#installing-additional-python-dependencies then I actually get `Pillow==8.1.2`.

So should we choose 8.1.2 ?

Reply:
I changed to 8.1.2